domain/repository_test.go

```diff
@@ -2,6 +2,7 @@ package domain
 
 import (
 	"github.com/stretchr/testify/assert"
+	"net/http"
```

Comment:
nit: it's a good practice to sort out imports. See https://vyacheslavpryimak.medium.com/golang-imports-sorting-grouping-hell-and-painless-tool-to-solve-it-6c8448d1fdf7

Reply:
Thanks for the advice